### PR TITLE
pm2: update page

### PR DESCRIPTION
**Pull request by nire0510**
New commands: save, resurrect & monit

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pm2.md [hunk 1]
```diff
@@ -23,3 +23,15 @@
 - Restart a process:
 
 `pm2 restart {{myapp}}`
+
+- Dump all processes for resurrecting them later:
```
Comment (sbrl) [inline pages/common/pm2.md]: This doesn't make grammatical sense. Perhaps we could do something like this instead?
```suggestion
- Dump all processes for later resurrecting:
```
Also, off choice of words. Usually commands like this use freeze / thaw or something like that instead. "Resurrecting" makes it sound like the process has died previously, which doesn't appear to technically be the case here.

Comment (nire0510) [reply pages/common/pm2.md]: Hi @sbrl , I don't mind accepting your suggestion, but I actually took the description from their own help file, **as is** 🤔. What do you suggest?

Comment (agnivade) [reply pages/common/pm2.md]: We usually prefer to change the descriptions to make it more suitable to newcomers. But I am curious to know @sbrl why do you think this is grammatically incorrect. It looks fine to me, or perhaps "... to resurrect them later" is also okay.

Comment (sbrl) [reply pages/common/pm2.md]: Taking another look, it does actually make sense here. It still sounds weird though - but I guess this is fine as-is.

**Comment by mebeim** [comment]
Merging since previous comments have all been addressed.

**Comment by mebeim** [comment]
Ah, probably should have changed the commit message to be clearer. My bad :\

**Pull request merged**